.github/actions/update-infrastructure/action.yml

```
@@ -96,7 +99,11 @@ runs:
       shell: pwsh
       run: |
         az config set bicep.use_binary_from_path=false
+<<<<<<< HEAD
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)